packages/istanbul-lib-source-maps/lib/transformer.js

```diff
@@ -257,9 +265,23 @@ SourceMapTransformer.prototype.transform = function(coverageMap) {
             debug('File [' + file + '] ignored, nothing could be mapped');
         }
     });
-    return libCoverage.createCoverageMap(output);
+    return libCoverage.createCoverageMap(getOutput(uniqueFiles));
 };
 
+function getUniqueKey(path) {
+    return path.replace(/[\\/]/g, '_');
+}
+
+function getOutput(cache) {
+    var output = {},
+        item;
+    Object.keys(cache).forEach(function(key) {
```

Comment:
nit: this could be `Object.keys(cache).reduce(...)` and I think it would read a lot nicer.

Reply:
I'm not sure I'm following here, the code loops over the `cache` keys to populate the `output` object with data from each cached item, it's not accumulating a single sum?

Reply:
It could look like this:

```js
function getOutput(cache) {
  return Object.keys(cache).reduce((output, key) => {
    const item = cache[key];
    return Object.assign(output, {
      [item.file]: item.mappedCoverage
    });
  }, {});
}
```

If you think about it, it is accumulating (or transforming, or "reducing") the values of the cache object to another object. So really everything can be a reduce 😉 but like I said, this is just a stylistic nit.

I do want the `var output, item` comma declaration changed, I'm surprised prettier didn't catch that!

Reply:
Done!